frontend/src/components/EditTripComponents/EditAccommodationInfo.tsx

```diff
@@ -0,0 +1,44 @@
+import {Accommodation} from "../../model/Trip";
+import {ChangeEvent} from "react";
+
+type EditAccommodationInfoProps = {
+    accommodations: Accommodation[]
+    setAccommodations: (value: Accommodation[]) => void
+}
+
+export default function EditAccommodationInfo({accommodations, setAccommodations}: EditAccommodationInfoProps) {
+
+    const handleAccommodationFormChange = (event: ChangeEvent<HTMLSelectElement>, index: number) => {
+        let data: Accommodation[] = [...accommodations];
+
+        // @ts-ignore
+        data[index][event.target.name] = (event.target.value);
+        console.log(data);
+        setAccommodations(data);
+    }
+    return (
+
+        <div className={"edit-trip-input-overview"}>
+            {accommodations.map((accommodationInput, index) => {
+                return (
```

Comment:
Warum ist hier ein zweites/genestetes return statement?

Reply:
Das hängt mit meinem Datenmodel zusammen. Die verschiedenen Emissions werden in Listen gespeichert um besser erweitert werden zu können, Dieses genestete Return ermöglicht die die Ausgabe der einzelnen Listen Elemente.